test/interactions/keyInteractionTests.ts

```diff
@@ -230,7 +258,7 @@ describe("Interactions", () => {
         svg.remove();
       });
 
-      it("removing keyRelease callbacks is possible", () => {
+      it("can remove removing keyRelease callbacks", () => {
         keyInteraction.onKeyRelease(aCode, aCallback);
```

Comment:
That is being asserted in the previous "it". Does it make sense to only assert it here?